rpm/osc.spec

```diff
@@ -6,44 +6,43 @@
 
 Name:           osc
 Summary:        Open Build Service Commander
-Version:        0.168.2
+Version:        0.177.0
 Release:        0
 License:        GPLv2+
 URL:            https://github.com/openSUSE/osc
 BuildArch:      noarch
 Source0:        %{name}-%{version}.tar.xz
 #For SailfishOS
-Patch0201:      0201-Add-sb2install-support-to-osc.patch
-Patch0202:      0202-Support-osc-copyprj-in-api-by-Islam-Amer.patch
-Patch0203:      0203-Support-synchronous-copyproj.patch
-Patch0204:      0204-Add-p-to-copyprj-to-enable-copying-of-prjconf.patch
-Patch0205:      0205-Add-support-for-rebuild-and-chroot-only-in-build.patch
-Patch0206:      0206-Add-architecture-and-scheduler-maps.patch
-Patch0207:      0207-Trap-any-kind-of-exception-during-plugin-parsing-eg-.patch
-Patch0208:      0208-Fixup-old-style-print-statements.patch
+# $  i=1; for j in 0*patch; do printf "Patch%04d: %s\n" $i $j; i=$((i+1));done
+Patch0001: 0001-Add-sb2install-support-to-osc.patch
+Patch0002: 0002-Support-osc-copyprj-in-api-by-Islam-Amer-usage-osc-c.patch
+Patch0003: 0003-Support-synchronous-copyproj.patch
+Patch0004: 0004-Add-p-to-copyprj-to-enable-copying-of-prjconf.patch
+Patch0005: 0005-Add-support-for-rebuild-and-chroot-only-in-build.-re.patch
+Patch0006: 0006-Add-architecture-and-scheduler-maps.patch
+Patch0007: 0007-Trap-any-kind-of-exception-during-plugin-parsing-eg-.patch
 
-BuildRequires:  python3-devel	
+BuildRequires:  python3-devel
 BuildRequires:  python3-distro
 BuildRequires:  rpm-python
-Requires:       python3-distro
-Requires:       rpm-python
-Requires:       python-m2crypto
-Requires:       python3-lxml
```

Comment:
Hmm, why Requires: -> BuildRequires:? Now at least python3-distro and rpm-python are duplicate lines...

Reply:
Because the python dependency scanner should pick up the Requires. I'll ix the duplicates.